django_project/base/models/project.py

```diff
@@ -128,7 +130,7 @@ def save(self, *args, **kwargs):
         if not self.pk:
             words = self.name.split()
             filtered_words = [t for t in words if t.lower() not in STOP_WORDS]
-            new_list = unicode(' '.join(filtered_words))
+            new_list = unidecode(' '.join(filtered_words))
```

Comment:
If it's not obvious what it does, you can also put some comment here. It will help someone in the future.